eng/Publishing.props

```diff
@@ -6,239 +6,29 @@
-      PublishFlatContainer="true"
-      AzureDevOpsCollectionUri="$(SYSTEM_TEAMFOUNDATIONCOLLECTIONURI)"
-      AzureDevOpsProject="$(SYSTEM_TEAMPROJECT)"
-      AzureDevOpsBuildId="$(BUILD_BUILDID)"
-      ItemsToPush="@(SdkAssetsToPushToBlobFeed);@(ChecksumsToPushToBlobFeed)"
-      ItemsToSign="@(ItemsToSignPostBuild)"
-      StrongNameSignInfo="@(StrongNameSignInfo)"
-      CertificatesSignInfo="@(CertificatesSignInfo)"
-      FileSignInfo="@(FileSignInfo)"
-      FileExtensionSignInfo="@(FileExtensionSignInfo)"
-      ManifestBuildData="@(ManifestBuildData)"
-      ManifestRepoUri="$(BUILD_REPOSITORY_NAME)"
-      ManifestBranch="$(BUILD_SOURCEBRANCH)"
-      ManifestBuildId="$(BUILD_BUILDNUMBER)"
-      ManifestCommit="$(BUILD_SOURCEVERSION)"
-      ManifestRepoOrigin="$(GitHubRepositoryName)"
-      IsStableBuild="$(IsStableBuild)"
-      PublishingVersion="$(PublishingVersion)"
-      AssetManifestPath="$(InstallersAssetManifestFilePath)"
-      IsReleaseOnlyPackageVersion="$(IsReleaseOnlyPackageVersion)"
-      PushToLocalStorage="$(PushToLocalStorage)"
-      AssetsLocalStorageDir="$(SourceBuiltAssetsDir)"
-      ShippingPackagesLocalStorageDir="$(SourceBuiltShippingPackagesDir)"
-      NonShippingPackagesLocalStorageDir="$(SourceBuiltNonShippingPackagesDir)"
-      AssetManifestsLocalStorageDir="$(SourceBuiltAssetManifestsDir)" />
-  </Target>
-
+  <Choose>
+    <When Condition="'$(EnableDefaultArtifacts)' == 'true'">
+      <PropertyGroup>
+        <!-- FSharp needs to push different packages to nuget.org depending on whether the SDK is preview or not,
+            To achieve this, we find the FSharp compiler package, then the stable or non-stable FSharp.Core and Compiler service
+            package contained within, depending on the stability switch of the SDK. The SDK then treats these packages as its own outputs,
+            which means they get automatically pushed on release day. -->
+        <FSharpCorePath Condition="'$(DotNetFinalVersionKind)' != 'release'">Shipping</FSharpCorePath>
+        <FSharpCorePath Condition="'$(DotNetFinalVersionKind)' == 'release'">Release</FSharpCorePath>
+      </PropertyGroup>
+      <ItemGroup>
+        <Artifact Include="$(NuGetPackageRoot)\Microsoft.FSharp.Compiler\$(MicrosoftFSharpCompilerPackageVersion)\contentFiles\$(FSharpCorePath)\FSharp.Core.*.nupkg
+                           $(NuGetPackageRoot)\Microsoft.FSharp.Compiler\$(MicrosoftFSharpCompilerPackageVersion)\contentFiles\$(FSharpCorePath)\FSharp.Compiler.Service.*.nupkg"
+                  PublishFlatContainer="false" />
+      </ItemGroup>
+    </When>
+    <Otherwise>
+      <ItemGroup>
+        <Artifact Include="$(ArtifactsPackagesDir)**\VS.Tools.*.nupkg;
+                          $(ArtifactsPackagesDir)**\VS.Redist.*.nupkg"
+                  IsShipping="$([System.String]::Copy('%(RecursiveDir)').StartsWith('Shipping'))"
+                  PublishFlatContainer="false" />
+      </ItemGroup>
+    </Otherwise>
```

Comment:
We have this in Signing.props in windowsdesktop: https://github.com/dotnet/windowsdesktop/blob/d4d4140397e0782287ac597d9609afd06d69d48e/eng/Signing.props#L26-L32

Reply:
I'll move it there.